src/Tizen.Network.Stc/Tizen.Network.Stc.csproj

```diff
@@ -0,0 +1,12 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="..\Tizen\Tizen.csproj" />
```

Comment:
Use ProjectReference instead of PackageReference

Reply:
done.